tests/test_slurm_system.py

```diff
@@ -48,6 +54,20 @@ def test_parse_sinfo_output(slurm_system):
     slurm_system.parse_sinfo_output(sinfo_output, node_user_map)
     assert slurm_system.partitions["main"][0].state == SlurmNodeState.IDLE
     assert slurm_system.partitions["main"][1].state == SlurmNodeState.IDLE
+    
+def test_parse_sinfo_output2(slurm_system):
```

Comment:
Let's use more descriptive function names for test cases as well. Names like test1, test2, and test3 are not good choices.

Reply:
the old one was deleted now